feature-interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt

```diff
@@ -164,3 +175,99 @@ private fun InterestsContent(
 private fun InterestsEmptyScreen() {
     Text(text = stringResource(id = R.string.interests_empty_header))
 }
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Preview(name = "phone", device = "spec:shape=Normal,width=360,height=640,unit=dp,dpi=480")
+@Preview(name = "landscape", device = "spec:shape=Normal,width=640,height=360,unit=dp,dpi=480")
+@Preview(name = "foldable", device = "spec:shape=Normal,width=673,height=841,unit=dp,dpi=480")
+@Preview(name = "tablet", device = "spec:shape=Normal,width=1280,height=800,unit=dp,dpi=480")
+@Composable
+fun InterestsScreenPopulated() {
+    NiaTheme {
+        NiaBackground {
+            Scaffold(
+                containerColor = Color.Transparent,
+                contentColor = MaterialTheme.colorScheme.onBackground,
+            ) { padding ->
+                InterestsScreen(
+                    uiState = InterestsUiState.Interests(
+                        authors = previewAuthors.map { FollowableAuthor(it, false) },
+                        topics = previewTopics.map { FollowableTopic(it, false) }
+                    ),
+                    tabState = InterestsTabState(
+                        titles = listOf(R.string.interests_topics, R.string.interests_people),
+                        currentIndex = 0
+                    ),
+                    followAuthor = { _, _ -> },
+                    followTopic = { _, _ -> },
+                    navigateToAuthor = {},
+                    navigateToTopic = {},
+                    switchTab = {},
+                    modifier = Modifier.padding(padding)
+                )
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Preview(name = "phone", device = "spec:shape=Normal,width=360,height=640,unit=dp,dpi=480")
+@Preview(name = "landscape", device = "spec:shape=Normal,width=640,height=360,unit=dp,dpi=480")
+@Preview(name = "foldable", device = "spec:shape=Normal,width=673,height=841,unit=dp,dpi=480")
+@Preview(name = "tablet", device = "spec:shape=Normal,width=1280,height=800,unit=dp,dpi=480")
+@Composable
+fun InterestsScreenLoading() {
+    NiaTheme {
+        NiaBackground {
+            Scaffold(
```

Comment:
I'm curious why these previews are being wrapped in a `Scaffold`? I don't see an obvious reason to have it.

Reply:
Hmm I thought it was needed.. But you're right, it doesn't make a difference. Removed them!